cortex-certifai-examples.gocd.yaml

```diff
@@ -47,6 +47,8 @@ common:
               mkdir -p artifacts
               cp -f certifai_toolkit.zip artifacts/
               c12e-ci
+              pip install jinja2 requirements-parser
+              ./build.sh local-docker
```

Comment:
The `c12e-ci` step runs the examples in the pipeline, but it previously wasn't building the base images for prediction service templates. I've added the step to perform build step as a sanity check